homeassistant/components/ring/__init__.py

```diff
@@ -138,3 +133,28 @@ def _async_migrator(entity_entry: er.RegistryEntry) -> dict[str, str] | None:
         return None
 
     await er.async_migrate_entries(hass, entry_id, _async_migrator)
+
+
+async def async_migrate_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
+    """Migrate old config entry."""
+    entry_version = entry.version
+    entry_minor_version = entry.minor_version
+
+    new_minor_version = 2
+    if (
+        entry_version == 1
+        and entry_minor_version < new_minor_version <= CONF_CONFIG_ENTRY_MINOR_VERSION
```

Comment:
```suggestion
        and entry_minor_version == 2
```
Rather add it like this

Reply:
That done but to `entry_minor_version == 1`